python/heterocl/tvm/runtime.py

```diff
@@ -1,7 +1,8 @@
 from ._ffi.function import register_func
 import os, subprocess, time, re, glob
 from ..report import parse_xml
-debug = False
+from ..devices import Project
```

Comment:
Where is this used?

Reply:
The project class? It's used in the HLS resources report function to store the work directory path.

Reply:
Nvm. It seems github is not showing the right changes.